deploy/bundle/manifests/backingstore.crd.yaml

```diff
@@ -30,10 +30,14 @@ spec:
         description: BackingStore is the Schema for the backingstores API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
snapshotter changes look good to me but these are unrelated to this PR. do we need a separate PR for this one?

Reply:
This is automatic after running `make gen-latest-csv`

Reply:
This is also not required after latest commits.